djstripe/contrib/rest_framework/serializers.py

```diff
@@ -28,7 +28,8 @@ class CreateSubscriptionSerializer(serializers.Serializer):
 
     stripe_token = serializers.CharField(max_length=200)
     plan = serializers.CharField(max_length=50)
-    charge_immediately = serializers.NullBooleanField(required=False)
+    charge_immediately = serializers.BooleanField(required=False, allow_null=True, default=None)
+    charge_immediately.null = True
```

Comment:
this is being re-considered at https://github.com/dj-stripe/dj-stripe/pull/1251#discussion_r507919489